### DISCUSSION TITLE
Enable Glue (Reversible) MIDI?

### DISCUSSION THREAD

**Discussion by MonkeyBars3k** [MonkeyBars3k/ReaScripts#63] [Ideas]
Implement real MIDI reversible gluing as opposed to rendered virtual instrument

**Comment by basicchannels** [comment]
What do you mean by this exactly? Do you mean so it stays as midi data? And if two midi items are glued they are glued together but remain midi? 

I think that would be cool, but not a massive thing for me. I already take the gluing audio and midi together as a super awesome but ultimately a bit gimmicky feature, because in reality I so seldomly have audio and midi working on the same track.

I DO HOWEVER, think it raises a good point, but perhaps more thought of as Glue Reversible (MIDI) and that's it.

It makes a lot more sense in my head to just have the same feature but directed just to midi, without worrying about the combining midi and audio stuff

**Comment by MonkeyBars3k** [comment]
Good points. This leads me to considering a explicitly audio-only approach other than the current render MIDI through VI feature. Like to the point of calling the scripts "Glue (Reversible) audio" etc

**Comment by MonkeyBars3k** [comment]
Otoh, reversible gluing of MIDI is still pretty useful, and not included as a native feature, despite pooling being supported. Moving card to discussion though
